src/pages/groups/Group.js

```diff
@@ -17,6 +17,7 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 import { Component } from 'src/libs/wrapped-components'
 import { styles } from 'src/pages/groups/common'
+import { LargeFadeBox } from 'src/pages/workspaces/List'
```

Comment:
Now that this is becoming a shared component, we should move it to a more central location. `components/common.js` would be a good choice. Also, we should pause and consider the name. Is there anything inherently 'large' about this, or could it just be `FadeBox`? Is this suitable for the Browse Data page as well?

Reply:
I agree about moving it to common.js
I called it large fade box because while it is similar looking to FadeBox from Browse Data, I had to change quite a few style elements to make it fit for a whole page rather than just the smaller element, so I thought it warranted its own method

Reply:
We could rename it to something like FullPageFadeBox so its clearer? Although that name is kind of clunky...